packages/ffe-cards/less/components.less

```diff
@@ -8,16 +8,20 @@
     }
 
     &--subtext {
+        &:extend(.ffe-small-text);
+
         margin: 5px 0 0 0;
-        color: @ffe-grey-charcoal;
     }
 
     &--title {
         &:extend(.ffe-h5);
 
         margin: 0;
-        text-overflow: ellipsis;
-        white-space: nowrap;
-        overflow: hidden;
+
+        &--overflow-ellipsis {
+            text-overflow: ellipsis;
+            white-space: nowrap;
+            overflow: hidden;
+        }
```

Comment:
Unfortunately this calls for a major version bump. An alternative is to make a logically reversed modifier, but this would add more styling, and also I am under the impression that we prefer to not make no-something-something modifiers.

Any opinions on this?

Reply:
I think it's better to eat that new major now that going down the not-no path

Reply:
I agree, this is a better approach than a modifier to reset the ellipsis.